addon/components/frost-table.js

```diff
@@ -54,12 +81,34 @@ export default Component.extend({
     return columns.map((column, index) => Object.assign({index}, column))
   },
 
+  @readOnly
+  @computed()
+  _isSelectable () {
+    return !isNone(this.get('onSelectionChange'))
+  },
+
   // == Functions =============================================================
 
+  setShift (event) {
+    if (!this.isDestroyed) {
+      this.set('_isShiftDown', event.shiftKey)
+    }
+  },
+
   // == DOM Events ============================================================
 
   // == Lifecycle Hooks =======================================================
 
+  init () {
+    this._super(...arguments)
+    const itemKey = this.get('itemKey')
+    if (itemKey) {
+      this.set('_itemComparator', function (lhs, rhs) {
```

Comment:
`_itemComparator` will never be set if itemKey is missing. I saw `itemKey` is assigned a default value inside `frost-table-row`. It's probably better to move the logic here and pass `itemKey` down.